pinot-common/src/main/java/org/apache/pinot/common/function/scalar/DataTypeConversionFunctions.java

```diff
@@ -43,6 +43,7 @@ private DataTypeConversionFunctions() {
   public static Object cast(Object value, String targetTypeLiteral) {
     try {
       Class<?> clazz = value.getClass();
+      // TODO: Support cast for MV
```

Comment:
did you meant element-wise? how would the semantic work?

Reply:
Yes. It is supported in the transform function, but not in the scalar function